src/Core/Enums/CipherRepromptType.cs

```diff
@@ -0,0 +1,8 @@
+﻿namespace Bit.Core.Enums
+{
+    public enum CipherRepromptType: byte
```

Comment:
I know Kyle mentioned the name (to remove "password" from it), and now it's my turn to nit on the name 😏  (sorry), but perhaps `CipherAccessRepromptType`, or just `RepromptType`, otherwise I'm fine with this as-is, but it seems like while this property will be tied to a `Cipher`, it's prompting the cipher (it's re-prompting to access the cipher)

Reply:
Nevermind, we'll keep this name 😁 , sorry.